src/components/event/EventItem.jsx

```diff
@@ -16,30 +16,37 @@ function Event ({ event, onLike }) {
     <Card className={classes}>
       <Card.Body>
         <Row>
-          <Col xs={8}>
+          <Col>
             <Link to={`/event/${event.id}`}>
               <h3>{event.title}</h3>
+              {event.organizers &&
+                <p>
+                  {event.organizers.map(organizer => (
+                    <span key={organizer.id}>{organizer.title}</span>
```

Comment:
Maybe we should add a link to the organizer details page here? 
Do we get (implicit) spacing between multiple organizers or should we add some?

Reply:
You are absolutely right. I've updated the PR